bee-message/src/output/extended.rs

```diff
@@ -95,17 +90,20 @@ pub struct ExtendedOutput {
 impl ExtendedOutput {
     /// The [`Output`](crate::output::Output) kind of an [`ExtendedOutput`].
     pub const KIND: u8 = 3;
-    ///
-    const ALLOWED_FEATURE_BLOCKS: [u8; 8] = [
-        SenderFeatureBlock::KIND,
-        DustDepositReturnFeatureBlock::KIND,
-        TimelockMilestoneIndexFeatureBlock::KIND,
-        TimelockUnixFeatureBlock::KIND,
-        ExpirationMilestoneIndexFeatureBlock::KIND,
-        ExpirationUnixFeatureBlock::KIND,
-        MetadataFeatureBlock::KIND,
-        IndexationFeatureBlock::KIND,
-    ];
+
+    /// Returns the set of allowed [`FeatureBlock`]s for an [`ExtendedOutput`].
+    #[inline(always)]
+    pub fn allowed_feature_blocks() -> FeatureBlockUsages {
+        FeatureBlockUsages::empty()
```

Comment:
Is the empty necessary ?

Reply:
It's not necessary, but I like it for readability reasons. I can remove it though, your call.